### DISCUSSION TITLE
p/debuginn-2023/

### DISCUSSION THREAD

**Discussion by giscus** [debuginn/blog#238] [General]
# p/debuginn-2023/

工作 今年工作经历了比较大的调整，由于部门技术栈逐渐趋于统一从 Go 技术栈转换到 Java 技术栈，也就是说我会双语言啦，自己也开始从事 Java 相关业务的开发，其实在 22 年底就开始接触 Java 服务了，按照学习计划，其实切换起来，还是相对来说比较简单，也没有太大的阻力，但是随着深入学习，发现自己学习的知识还是太少了，只能是 Java 庞大生态的冰山一角，值得自己骄傲的是，这一年，在 leader 和团队同事的支持和帮助下，我从 0 到 1 实现了一套并发框架，命名为 PhoenixCore，目前已经应用在了小米商城产品站（也就是说，只要你访问过小米商城，这套框架就为你提供过服务 😊），年后我会逐一整理并发表一系列的围绕并发框架的系列文章，来讲解我是如何实现这个框架以及我遇到的相关问题及解决思路。
站点 今年，站点最大的变化就是完成了对站点的静态化建设，可以将重点放在博客文章的输出中去，同时省去了自己维护站点的大量精力。目前站点的运营模式就是 Github 来提供文章的编写及打包发布，使用 Github pages 部署，CDN 使用免费的 Cloudflare 和 jsDelivr，（这也就代表着网站可能会出现访问不稳定 😛），同时将站点域名从 debuginn.cn 转换为 debuginn.com。
今年由于站点域名和子域名的调整，统计开始包含自己的子域名访问记录，由于期间有部分访问记录丢失，环比指标都有下降，不过随着自己开始专注于文章的输出，情况开始变好。
由于自己重新调整了博客链接组织结构，404 的页面一直占据着很高的比例 😂。
文章 今年输出并不是很多，还是和去年一样，都一一陈列一下吧。
2023 年为何我还在使用 RSS 谈谈 ChatGPT 如何接手并维护一个项目 全站静态化升级完毕 FAAS 调研笔记 长桥港美股团办活动 希望我的文章可以带给你帮助，希望我和大家一起变得更强～
开源 今年由于自己博客迁移到了 Github，所以说活跃多了起来 🐶。
生活 23年去了很多地方旅游，云南（昆明、大理、丽江、香格里拉）、甘肃、天津、河北（张家口）、山东（济宁、威海、青岛）。
3月份去奥森桃花谷看了桃花，人很多、花更美； 4月份和自己的爱人一起去了云南，喂了滇池的海鸥、感受到了苍山洱海的美、看到了大冰的小屋、看到了雄伟的玉龙雪山、体验到了香格里拉藏区人们的淳朴与善良好客； 5月份去了甘肃老家，感受到了大西北的豪情、辽阔，看到了一望无际的荒漠与祁连山脉，看到了《隐入尘烟》中甘肃特色的土房子，品尝到了特色的面卷子与羔羊； 同 5月去了环球影城，感受到了电影世界。去北动看“西直门三台子”萌兰，很遗憾天太热，没看到，只看到了萌大和其他熊猫； 6月去了天津，看了民国风的五大道，听了场相声，还品尝了特色小吃； 7月陪着姐姐姐夫大外甥在北京城逛了一番，很快乐； 10月回了趟家，之后去威海，自驾到了山东最东端 - 始皇东巡处，看到了搁浅的“布鲁维斯号”，去青岛的海鲜市场，买了海鲜，赶了海。 23年，也一直坚持着徒步，参加了 4次活动，其中 2次夜爬活动，看到了星空、看到了日出。

https://blog.debuginn.com/p/debuginn-2023/

**Comment by debuginn** [comment]
2024，新年快乐～

**Comment by wangdefou-dev** [comment]
新年快乐~
祝愿2024年快乐幸福

**Reply by debuginn** [reply]
谢谢 愿 24年 大家一起变得更强～

**Reply by wangdefou-dev** [reply]
加油加油！
